tests/test_api.py

```diff
@@ -18,18 +18,25 @@ def _sign(self, data: dict):
 
     def test_no_data(self):
         """Fail properly when no data is sent"""
-        r = self.client.post("/sign")
+        r = self._sign(None)
         self.assertEqual(400, r.status_code, r.data)
         self.assertIn(b"Missing request body", r.data)
 
     def test_no_csr(self):
         """Fail properly if no csr is provided"""
         r = self._sign({"data": 12})
         self.assertEqual(400, r.status_code, r.data)
-        self.assertIn(b"Missing required field 'csr'", r.data)
+        self.assertIn(b"Missing required field &#39;csr&#39;", r.data)
```

Comment:
I couldn't find a better way to handle apostrophes in assertIn (escaping it didn't work)
Any suggestions are welcome